pkgs/tools/graphics/grimshot/default.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenv
+, fetchFromGitHub
+, sway-unwrapped
+, wl-clipboard
+, libnotify
+, slurp
+, grim
+, jq
+}:
+
+stdenv.mkDerivation {
+  pname = "grimshot";
+  version = "2020-05-08";
+
+  # after sway-1.5 this may be switched to sway-unwrapped.src
+  src = fetchFromGitHub {
+    owner = "swaywm";
+    repo = "sway";
+    # latest change to contrib/grimshot*
+    rev = "b1d08db5f5112ab562f89564825e3e791b0682c4";
+    sha256 = "1j9wk2m79grhjxyx8fzgcza0virlkansl2fvlvjszk526i0igri6";
+  };
```

Comment:
Are there major issues with grimshot from Sway 1.4?

Reply:
there were a bunch of improvements 2 weeks ago
a man page was also added

Reply:
That's a bit unfortunate, seems like there are too many commits to cherry-pick them into `sway-unwrapped`. But ideally we'd still use 1.4 then as long as it works.

Reply:
i'd really miss the "manually select window to screenshot" function
and i think the man page is a really nice thing
ofc, if 1.5 gets released we can switch to that

Reply:
i suppose i could switch to downloading just the 2 files i'm using

Reply:
done